Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -108,6 +108,13 @@ extension SPNoteEditorViewController {
         interlinkProcessor.delegate = self
         interlinkProcessor.contextProvider = self
     }
+    
+    /// Sets up the Keyboard
+    ///
+    @objc
+    func configureTextViewKeyboard() {
+        noteEditorTextView?.keyboardDismissMode = UIScrollView.KeyboardDismissMode.interactive
```

Comment:
Swift's Type Inference allows you to do just this:

```
noteEditorTextView?.keyboardDismissMode = .interactive
```

Reply:
Ah cool!  Updated.  Will ship once CI checks are complete